lib/cisco_node_utils/cmd_ref/system.yaml

```diff
@@ -1,8 +1,7 @@
 # system
 ---
 resources:
-  nexus:
-    get_command: "show system resources"
-  # ios_xr:
-    # grpc doesn't allow this command
-    # get_command: 'monitor threads iteration 1'
+  # grpc doesn't allow this command
+  # get_command: 'monitor threads iteration 1'
```

Comment:
It was okay before, it's better now. :-)

Kidding... I was troubleshooting one of the tests and just changed it since I prefer the explicit _exclude over the implicit "unsupported". But I'll change it back if you'd prefer.